wagtailio/project_styleguide/templates/patterns/pages/home/home_page.html

```diff
@@ -0,0 +1,47 @@
+{% extends "patterns/base_page.html" %}
+
+{% load static wagtailcore_tags wagtailembeds_tags %}
+
+{% block body_class %}template-homepage{% endblock %}
+
+{% block content %}
+
+    {% include "patterns/components/hero/hero.html" with value=page classes="hero--home" %}
+
+    <div class="grid">
+        <div class="code-snippet{% if not page.cta_one or not page.cta_two %} code-snippet--bottom-space{% endif %}" data-code-snippet>
+            <pre data-code-to-copy>pip install wagtail</pre>
+            <button class="code-snippet__button" aria-label="Copy the command pip install wagtail" data-copy-code-button>
+                <svg class="code-snippet__icon" aria-hidden="true">
+                    <use xlink:href="#copy"></use>
+                </svg>
+            </button>
+            <p class="code-snippet__confirmation">Copied!</p>
+        </div>
+    </div>
+
+    {% if page.cta_one or page.cta_two %}
+        <div class="grid">
+            <div class="button-align button-align--home">
+                {% if page.cta_one %}
```

Comment:
Note: I would think these would be a CTA streamfield?

Reply:
I'm not sure they'd be a streamfield as they're specific to the hero?